main.go

```
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"btc-app/config"
+	"btc-app/server"
+	"github.com/joho/godotenv"
+)
+
+func main() {
+	_ = godotenv.Load()
```

The current code ignores any error returned. It's a good practice to handle errors
```
if err := godotenv.Load(); err != nil {
   ///handle error
}
```

---

In case if env variables couldn't be loaded -> program will failed.